extras/kubernetes/heketi-deployment.json

```diff
@@ -0,0 +1,160 @@
+{
+  "kind": "Service",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "heketi",
+    "labels": {
+      "glusterfs": "heketi-service",
+      "deploy-heketi": "support"
+    },
+    "annotations": {
+      "description": "Exposes Heketi Service"
+    }
+  },
+  "spec": {
+    "selector": {
+        "name": "heketi"
+    },
+    "ports": [
+      {
+        "name": "heketi",
+        "port": 8080,
+        "targetPort": 8080
+      }
+    ]
+  }
+}
+{
+    "kind": "Deployment",
+    "apiVersion": "extensions/v1beta1",
+    "metadata": {
+        "name": "heketi",
+        "labels": {
+            "glusterfs": "heketi-deployment"
+        },
+        "annotations": {
+            "description": "Defines how to deploy Heketi"
+        }
+    },
+    "spec": {
+        "replicas":1,
+        "template": {
+            "metadata": {
+                "name": "heketi",
+                "labels": {
+                    "name": "heketi",
+                    "glusterfs": "heketi-pod"
+                }
+            },
+            "spec": {
+                "containers": [
+                    {
+                        "image": "heketi/heketi:dev",
+                        "imagePullPolicy": "Always",
+                        "name": "heketi",
+                        "env": [
+                          {
+                            "name": "HEKETI_USER_KEY",
+                            "value": "${HEKETI_USER_KEY}"
```

Comment:
I have kept it like that for now so that user can add(manually edit) their values while deploying. I thought of mentioning in the documentation. I understand we can not pass environmental variable.